homeassistant/components/tts/marytts.py

```diff
@@ -0,0 +1,127 @@
+"""
+Support for the MaryTTS service.
+
+For more details about this component, please refer to the documentation at
+https://home-assistant.io/components/tts/marytts/
+"""
+import asyncio
+import logging
+import re
+
+import aiohttp
+import async_timeout
+import voluptuous as vol
+
+from homeassistant.components.tts import Provider, PLATFORM_SCHEMA, CONF_LANG
+from homeassistant.helpers.aiohttp_client import async_get_clientsession
+import homeassistant.helpers.config_validation as cv
+
+_LOGGER = logging.getLogger(__name__)
+
+SUPPORT_LANGUAGES = [
+    'de', 'en-GB', 'en-US', 'fr', 'it', 'lb', 'ru', 'sv', 'te', 'tr'
+]
+
+SUPPORT_VOICES = [
+    'cmu-slt-hsmm'
+]
+
+SUPPORT_CODEC = [
+    'aiff', 'au', 'wav'
+]
+
+CONF_MARYTTS_HOST = 'marytts_host'
+CONF_MARYTTS_PORT = 'marytts_port'
```

Comment:
Please use port/host from `const.py` so we have the same habdling over all platform/component.